stable/grafana/templates/deployment.yaml

```diff
@@ -83,6 +83,14 @@ spec:
           resources:
 {{ toYaml .Values.server.resources | indent 12 }}
           volumeMounts:
+            {{- if .Values.provisioningDatasourcesFiles }}
+            - name: provisioning-volume 
+              mountPath: {{ default "/var/lib/grafana/provisioning" .Values.server.provisioningLocalPath }}/datasources
```

Comment:
Don't use the `default` function. Defaults come from `values.yaml` anyways which is better. Apply to all your changes.

Reply:
I figured this is common practice in the chart. So, let's leave it for now. Would be cool to get that fixed in a separate PR.